src/LibExpat.jl

```diff
@@ -298,7 +298,7 @@ function find{T<:AbstractString}(pd::ETree, path::T)
         end
     end
 
-    xp= Array(Tuple{Symbol,Any},0)
+    xp= Array{Tuple{Symbol,Any}}(0)
```

Comment:
Maybe it's just me, but I trip over the `(0)`.  I find it much clearer, that [] is the empty Vector ;) Besides, it's less typing :D